### ISSUE TITLE
Meta Finetune problem

### ISSUE THREAD

**Issue by yushengsu-thu** [alibaba/EasyTransfer#35]
Hello, 
I follow the Meta Fine-tune tutorial and fine-tuned the model.  (I use the command in bash)
However, I cannot find:
1. where is the test set (amazon)
2. how to predict (lost part in the tutorial)
Even though I try to use PAI-modelzoom, the names of parameters are far different so that hard to know how to predict still.

Could you please provide the (test set-amazon) and (bash file-predicting part) in the tutorial.

Thanks

**Comment by chywang** [comment]
For test set, see: https://atp-modelzoo-sh.oss-cn-shanghai.aliyuncs.com/tutorial/meta_finetune/amazon/amazon_test.tsv

After you finetune the model, find the ckpt that you generate. You can use the easytransfer_app command to do prediction (see readme of the project).

**Comment by yushengsu-thu** [comment]
The problem is I always occure errors such as, KeyError: 'sequence_length', even though I follow the easytransfer_app command (on readme). The readme didn't introduce these parameters; thus, I sincerely hope to get an whole instruction for meta-fine part.

**Comment by yushengsu-thu** [comment]
I modified the config file and  code to predict in the test set as shown below:

app = Application()
pred_reader = CSVReader(input_glob=app.predict_input_fp,
                        is_training=False,
                        input_schema=app.input_schema,
                        batch_size=app.predict_batch_size)

pred_writer = CSVWriter(output_glob=app.predict_output_fp,
                        output_schema=app.output_schema)

app.run_predict(reader=pred_reader, writer=pred_writer, 
                checkpoint_path=app.predict_checkpoint_path)
==============================================
NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key dense/bias not found in checkpoint
	 [[node save/RestoreV2 (defined at /home/suyusheng/miniconda2/envs/easytransfer/lib/python3.6/site-packages/easytransfer-0.1.4-py3.6.egg/easytransfer/engines/model.py:629)  = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_INT64], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
==============================================


I’ve ready used the easytransfer_app commands on the readme. However, it always has bugs and hard to trace due to the complex framework, and I cannot find some details in the README and tutorial. It’s really hard to follow the easytransfer to implement the meta-fine. Could you please released a bash file, which is including meta-finetune, fine-tune, and predict parts to me? 


Thanks,
